packages/eslint-plugin/docs/rules/no-useless-template-expression.mdx

```diff
@@ -11,15 +11,6 @@ import TabItem from '@theme/TabItem';
 
 This rule reports template literals that contain substitution expressions (also variously referred to as embedded expressions or string interpolations) that are unnecessary and can be simplified.
 
-:::info[Migration from `no-useless-template-literals`]
-
-This rule was formerly known as [`no-useless-template-literals`](./no-useless-template-literals.mdx).
-We encourage users to migrate to the new name, `no-useless-template-expression`, as the old name will be removed in a future major version of typescript-eslint.
-
-The new name is a drop-in replacement with identical functionality.
-
-:::
```

Comment:
[Docs] I'd say let's keep this here for a while. Folks might get confused about the rename.

```mdx
:::info[Migration from `no-useless-template-literals`]

This rule was formerly known as [`no-useless-template-literals`](./no-useless-template-literals.mdx).
The new name is a drop-in replacement with identical functionality.

:::
```

Having the old rule named here will help with folks searching the old name.

Reply:
Got it. Reverted removal in [af63225](https://github.com/typescript-eslint/typescript-eslint/pull/9207/commits/af6322528d0b8191c27125ca7a0227549747b20f).

Reply:
Oh @y-hsgw I meant to suggest updating the text in `no-useless-template-expression.mdx` too. The old name is now removed, so the docs comment is out of date.

Reply:
Sorry🙇‍♂️
I have corrected it in commit c7a4ba5.